ui/package.json

```diff
@@ -24,6 +24,7 @@
     "react-transform-catch-errors": "^1.0.2",
     "react-transform-hmr": "^1.0.4",
     "reactable": "^0.14.1",
+    "react-table": "^3.1.0",
```

Comment:
Apparently `5.0.0` is the latest release according to: https://www.npmjs.com/package/react-table, think we should upgrade to that?